extensions/protobuf/src/main/java/org/modelmapper/protobuf/ProtobufHelper.java

```diff
@@ -89,6 +93,19 @@ public static Method setter(Class<?> type, String field) throws NoSuchMethodExce
     throw new NoSuchMethodException(methodName);
   }
 
+  public static Method adder(Class<?> type, String field) throws NoSuchMethodException {
+    String methodName = "add" + formatMethodName(field);
+    for (Method method : type.getMethods()) {
+      if (isSetterForBuilder(method, methodName))

```

Comment:
I've been trying to serialize a list with protobuf/modelmapper and this test fails for primitives.

I have "repeated int32 numbers" in my proto file. And I'm trying to map that to a `List<Integer> numbers` within another object.

In the check, this case passes:
```
Class<?>[] parameterTypes = method.getParameterTypes();
       return parameterTypes.length == 1
```
But this check fails:

 `Builder.class.isAssignableFrom(parameterTypes[0])`

 because parameterTypes[0] is of type int, not of builder

I changed this check to be `isSetterForPrimitive` and it passed, because it's just the inverse case (looking for false). But I'm wondering if we need separate checks here, one for an object list and one for primitive lists?